modules/base/src/properties/valueaxisproperty.cpp

```diff
@@ -0,0 +1,152 @@
+/*********************************************************************************
+ *
+ * Inviwo - Interactive Visualization Workshop
+ *
+ * Copyright (c) 2025 Inviwo Foundation
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *********************************************************************************/
+
+#include <modules/base/properties/valueaxisproperty.h>
+
+#include <fmt/format.h>
+
+namespace inviwo {
+
+std::string_view ValueAxisProperty::getClassIdentifier() const { return classIdentifier; }
+
+ValueAxisProperty::ValueAxisProperty(std::string_view identifier, std::string_view displayName,
+                                     bool customAxis, InvalidationLevel invalidationLevel,
+                                     PropertySemantics semantics)
+    : CompositeProperty{identifier, displayName, invalidationLevel, semantics}
+    , customAxis_{customAxis}
+    , valueName_{"valueName", "Value name", ""}
```

Comment:
Might be enough with only Name/Unit due to the composite property name?
```suggestion
    , valueName_{"valueName", "Name", ""}
```

Reply:
I would stick with "Value name" to be explicit and in line with `DataRangeProperty`.

![image](https://github.com/user-attachments/assets/47eabd77-d092-4500-a511-4fcd7a4d342f)